styles/pronouns.page/catppuccin.user.css

```diff
@@ -219,6 +219,10 @@
       border-color: @accent-color !important;
     }
 
+    .form-control:focus {
+      box-shadow: 0 0 0 .25rem rgba(@accent-color, 0.5);
+    }
```

Comment:
Nesting please. Also, remember what I said about using `rgba` to change the opacity and using `%`? Use `fade(@accent-color, 5%)`. Use the feedback we've been giving you!